### PR TITLE
Pull request template added

### PR DESCRIPTION
**Pull request by Kunal-2001**
A pull request template for basic information

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+**Screenshots**\
+If applicable, include screenshots of the issue you solved.
+
+**Checklist**
+
+- [ ] I worked on a branch other than `main`.
+- [ ] My branch is up-to-date with the Upstream `main` branch.
+- [ ] I have fixed potential errors using `yarn lint`.
+- [ ] I ran `yarn build` to check if everything still builds successfully.
```
Comment (Panquesito7) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- [ ] I ran `yarn build` to check everything still builds successfully.
```

**Review by Panquesito7 — COMMENTED** [review]
👍 Good work.

**Comment by Kunal-2001** [comment]
> 👍 Good work.

Thanks

**Review by Panquesito7 — DISMISSED** [review]
LGTM. Thank you for your contribution! 👍 🎉

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!-- This will automatically close the issue number mentioned when merged in main branch -->
+
```
Comment (maltejur) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+Fixes #
+
+<!-- Description of the changes this pull request introduces -->
+
```
Comment (maltejur) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+**What change does this pull request introduce?**
+
+<!-- Screenshots of the final output -->
+
```
Comment (maltejur) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<!-- This will automatically close the issue number mentioned when merged in main branch -->
+
+Fixes #
```
Comment (maltejur) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
Closes #
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+<!-- This will automatically close the issue number mentioned when merged in main branch -->
```
Comment (maltejur) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
<!-- Add the number of the issue this pull request is closing here -->
```

**Review by maltejur — CHANGES_REQUESTED** [review]

**Review by maltejur — APPROVED** [review]

**Pull request merged**